src/Rule/Length.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Morebec\Validator\Rule;
+
+
+use InvalidArgumentException;
+use Morebec\Validator\ValidationRuleInterface;
+
+class Length implements ValidationRuleInterface
+{
+    /**
+     * @var int
+     */
+    private $length;
+    /**
+     * @var string|null
+     */
+    private $message;
+
+    /**
+     * Length constructor.
+     * @param int $length
+     * @param string|null $message
+     */
+    public function __construct(
+        int $length,
+        ?string $message = null
+    )
+    {
+        if($length <0)
+            throw new InvalidArgumentException();
```

Comment:
When throwing an exception, a message should always be provided in order to inform the user why the exception was thrown without needing to look at the source code.